### PR TITLE
JavaScript Capstone Project

### PR DESCRIPTION
**Pull request by kidde60**
## This is the final step in the app TV app. We have implemented the following:

- The home page.
- The comments popup.
- When the page loads, the webapp retrieves data from TV API and shows the list of items on screen.
- The Involvement API to show the item likes.
- When the user clicks on the Like button of an item, the interaction is recorded in the Involvement API and the screen is updated.
- When the user clicks on the "Comments" button, the Comments popup appears.
- Home page header and navigation similar to the given mockup.
- Home page footer similar to the given mockup.
- When the user clicks on the "Comment" button, the data is recorded in the Involvement API and the screen is updated.
- The number of items (home).
- The number of comments (comments popup).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+          ${movie.summary}
+        </div>
+          
+            <div class = "form" >
+              <label for="fname">Name:</label><br>
+              <input type="text" id="fname" name="fname" placeholder = "Enter your name"><br><br>
+              <label for="comment">Comment:</label><br>
+              <textarea name="comment" id="comment" cols="20" rows="3" placeholder = "Enter comment"></textarea>
+              <button id = ${id} class= "submit" type="submit"> Submit</button>
+            </div>
```
Comment (uwadonat) [inline src/index.js:155]: - [ ] Let's try to follow given mockup by adding title on comment add form and put comments before the form.

your design
Screenshot from 2022-09-11 00-04-26
Given design

given

File: README.md [hunk 2]
```diff
@@ -1,27 +1,24 @@
...
+- Major languages: HTML/CSS and JavaScript
+- Frameworks: Node.js 
+- APIs: IMDb API and Involvement API
 
-## Live Demo (if available)
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo]()
+[Vide Demo](./video/JSCapstone.mp4)
```
Comment (uwadonat) [inline README.md:18]: - [ ] The video should contains a highlight about a piece of the code that you find interesting or you are proud of making.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi **Team**

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good commit history ✔️
- Descriptive pull request message ✔️
- No linter errors ✔️
almoo


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey TEAM! 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**